### DISCUSSION TITLE
Saving configurations

### DISCUSSION THREAD

**Discussion by Shampoo-Denhut** [community/community#28236] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! First of all, I want to thank you for your product, I have to use it very often. Since I have a lot of ready-made configurations for different series, I have to manually copy the fields 'Input', 'Output' and the code field from notepad. I would very much like these settings to be saved in the program itself and substituted with the necessary data by one click. That would be great. Maybe it already exists and I just didn't find the right button?

I will be grateful for your feedback!

![2022-08-03_211022](https://user-images.githubusercontent.com/110561702/182679095-c2790525-b61a-4e7b-a6c6-649efbb7faa9.jpg)

**Comment by airtower-luna** [comment, marked as answer]
It looks like you're trying to report a bug in a software hosted on GitHub, maybe https://github.com/MattMcManis/Axiom? In that case you should open an issue *in that repository*. :slightly_smiling_face:

Though, if you're looking for something that can be automated, you might want to use [FFmpeg](https://ffmpeg.org/) directly without a GUI around it. That way you can script it.

**Reply by Shampoo-Denhut** [reply]
> It looks like you're trying to report a bug in a software hosted on GitHub, maybe https://github.com/MattMcManis/Axiom? In that case you should open an issue _in that repository_. 🙂
> 
> Though, if you're looking for something that can be automated, you might want to use [FFmpeg](https://ffmpeg.org/) directly without a GUI around it. That way you can script it.

Yes, exactly, thank you. I opened this topic in the wrong place. What should I do? Is it possible to reschedule or should I start it again?

**Reply by airtower-luna** [reply]
Just create a new issue there. You can probably reuse the screenshot link instead of uploading again. :smiley_cat:
